test/unittest/memory/memory_planner_validate.h

```diff
@@ -1,51 +0,0 @@
-// SPDX-License-Identifier: Apache-2.0
-/**
- * Copyright (C) 2021 Parichay Kapoor <pk.kapoor@samsung.com>
- *
- * @file   memory_planner_validate.h
- * @date   11 August 2021
- * @see    https://github.com/nnstreamer/nntrainer
- * @author Parichay Kapoor <pk.kapoor@samsung.com>
- * @bug    No known bugs except for NYI items
- */
-
-#ifndef __MEMORY_PLANNER_VALIDATE_H__
-#define __MEMORY_PLANNER_VALIDATE_H__
-
-#include <gtest/gtest.h>
-
-#include <memory>
-#include <string>
-
-#include <memory_planner.h>
-
-/**
- * @class MemoryPlannerValidate
- * @brief Memory planning validation
- */
-class MemoryPlannerValidate : public ::testing::TestWithParam<std::string> {
```

Comment:
One of the reason we make this Validate class separate is there could be multiple Memory Planner. So.. I think it might be better if we maintain as it is for later usage.

Reply:
The reason for moving this file to unittest_memory_planner_validate.cpp file is that it can hold all operation related to memory planner, using parameterized test. I think it doesnot need to divide file, which are used only for unittest_memory_planner.

Reply:
I see. Thanks.